python/kserve/kserve/storage/storage.py

```diff
@@ -152,6 +154,24 @@ def _update_with_storage_spec():
                     f.write(value)
                     f.flush()
 
+        if storage_secret_json.get("type", "") == "gs":
+            if storage_secret_json.get("base64_service_account", "") != "":
```

Comment:
@tjandy98  `base64_service_account` name is a bit confusing, from what I read this is a service account key file
https://cloud.google.com/iam/docs/keys-create-delete#iam-service-account-keys-create-console

Reply:
Thanks for your feedback. The idea is to support base64 encoded version of the credential. Do you have a suggestion?

Reply:
@yuzisun the idea is that the file is stored as a string secret and then mounted as a volume in GKE. Google's own docs do a similar thing: https://cloud.google.com/sql/docs/mysql/connect-kubernetes-engine#service-account-key-file

Reply:
Should we rename to base64_service_account_key_file ?